configure

```diff
@@ -542,7 +545,16 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros()
+  # use 'cc', as CC may be set to a target arch compiler command
+  # in case of cross-compilation
+  if sys.platform.startswith('aix'):
+    k = cc_macros('gcc')
+    matchup = {
+      '_ARCH_PPC'   : 'ppc',
+      '_ARCH_PPC64' : 'ppc64',
+    }
+  else:
+    k = cc_macros('cc')
```

Comment:
Actually I don't need to remove the else but instead just have it pass nothing in for that case.  Will go ahead and do that